rocky/account/views/account.py

```diff
@@ -13,30 +10,22 @@ class PageActions(Enum):
     WITHDRAW_ACCEPTANCE = "withdraw_acceptance"
 
 
-class AccountView(OrganizationView, DetailView):
-    template_name = "account_detail.html"
-    context_object_name = "member"
-
-    def get_object(self):
-        return OrganizationMember.objects.get(user=self.request.user, organization=self.organization)
-
+class OOIClearanceMixin(OrganizationView, TemplateView):
```

Comment:
A mixin should not inherit from View classes. So in this case `OOIClearanceMixin` should inherit from no other class. The `AccountView` and `OnboardingAcknowledgeClearanceLevelView` should inherit from  `OOIClearanceMixin`, `OrganizationView` and `TemplateView`. 

This will still inherit from two View classes which is also not correct, but was already the case and is something we should fix later. We still shouldn't make it worse however.